index.bs

```diff
@@ -489,24 +493,24 @@ When this method is invoked, the user agent MUST execute the following algorithm
 1. Let |issuedRequests| and |currentlyAvailableAuthenticators| be new [=ordered sets=].
 
 1. For each |authenticator| currently available on this platform, if
-    <code>{{options}}.{{ScopedCredentialOptions/attachment}}</code> is not [=present=] or its value matches |authenticator|'s
-    attachment modality, [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|.
+    <code>{{options}}.{{ScopedCredentialOptions/attachment}}</code> is [=present|not present=] or its value matches
+    |authenticator|'s attachment modality, [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|.
 
 1. [=set/For each=] |authenticator| in |currentlyAvailableAuthenticators|:
     1. Let |excludeList| be a new [=list=].
     1. [=list/For each=] credential |C| in <code>{{options}}.{{ScopedCredentialOptions/excludeList}}</code>:
-        1. If |C| has a non-empty {{transports}} list, and |authenticator| is connected over a transport not mentioned in
-            <code>|C|.{{transports}}</code>, the client MAY [=continue=].
+        1. If <code>|C|.{{transports}}</code> [=list/is not empty=], and |authenticator| is connected over a transport not
+            mentioned in <code>|C|.{{transports}}</code>, the client MAY [=continue=].
         1. Otherwise, [=list/Append=] |C| to |excludeList|.
     1. [=In parallel=], invoke the [=authenticatorMakeCredential=] operation on |authenticator| with |rpId|,
         |clientDataHash|, {{accountInformation}}, |normalizedParameters|, |excludeList| and |clientExtensions| as parameters.
     1. [=set/Append=] |authenticator| to |issuedRequests|.
 
 1. Let |promise| be [=a new promise=]. Return |promise| and start a timer for |adjustedTimeout| milliseconds. Then execute the
-    following steps [=in parallel=].
+    following steps [=in parallel=]. The [=task source=] for these [=tasks=] is the [=dom manipulation task source=].
```

Comment:
It seems to mean to me that if the user has many types of authenticators available, this could starve the dom manipulation task thread pool. Probably not a big deal, but may be an implementation quirk to handle.